io.github.niccokunzmann.python_dhcp_server.yml

```diff
@@ -0,0 +1,79 @@
+id: io.github.niccokunzmann.python_dhcp_server
+# The runtime is the environment that the application will run in, so you can use dependencies it provides
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# This is the command that will be run when the application is launched
+command: python_dhcp_server
+# These are the permissions that the application needs
+# Read more about finishing here: https://docs.flatpak.org/en/latest/manifests.html#finishing
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --persist=config
```

Comment:
you mean you write a `config` directory in the HOME directory of the user? You should fix that and follw the XDG directory specification I linked previously.

Reply:
I will remove this. It is not needed ... I used that during the time when I figured out what to do.

Reply:
0d23449